RealmWeaver/ModuleWeaver.cs

```diff
@@ -806,6 +826,74 @@ private TypeDefinition WeaveRealmObjectHelper(TypeDefinition realmObjectType, Me
         }
         helperType.Methods.Add(createInstance);
 
+        var copyToRealm = new MethodDefinition("CopyToRealm", MethodAttributes.Public | MethodAttributes.Final | MethodAttributes.HideBySig | MethodAttributes.Virtual | MethodAttributes.NewSlot, ModuleDefinition.TypeSystem.Void);
+        {
+            // This roughly translates to
+            /*
+				var casted = (ObjectType)instance;
+				
+				*foreach* non-list woven property in casted's schema
+				casted.Property = casted.Field;
+
+				*foreach* list woven property in casted's schema
+				var list = casted.field;
+				casted.field = null;
+				((ICopyValuesFrom)casted.Property).CopyValuesFrom(list);
```

Comment:
We should get rid of `ICopyValuesFrom` instead of making it public. Let's add a public `AddRange(IEnumerable<T>)` to `RealmList<T>` and call that instead.